server/public/model/config.go

```diff
@@ -967,6 +968,10 @@ func (s *ServiceSettings) SetDefaults(isUpdate bool) {
 	if s.EnableWebHubChannelIteration == nil {
 		s.EnableWebHubChannelIteration = NewPointer(false)
 	}
+
+	if s.FrameAncestors == nil {
+		s.FrameAncestors = NewPointer("")
```

Comment:
Let's make sure we capture this in the changelog, as `teams.microsoft.com` is no longer on by default.

Reply:
Done